index.html

```diff
@@ -24,6 +26,14 @@
       </ul>
     </nav>
   </div>
+  <script>

```

Comment:
jquery bad

Reply:
ouch dude :( wdym by bad?

Reply:
Erm well jquery is ok but it's a bit old and a bit too much. Raw JavaScript should be able to do the job, and if you want to type less you can assign characters to raw js functions, for example $ and $a to `querySelector` and `querySelectorAll` this way :
```
$ = document.querySelector.bind(document);
$a = document.querySelectorAll.bind(document);
```
I haven't looked into the code, though, so i don't know what uses jquery.